ui/pages/citizen.tsx

```diff
@@ -22,7 +17,6 @@ import AddToWallet from '../public/passport/wallet.svg'
 export default function Citizen() {
   const { data: account } = useAccount()
   const { data: passportData } = usePassport(account?.address)
```

Comment:
`usePassport` will break here, since you've removed from import

Reply:
usePassport has not been removed, works fine.